ztreamy/server.py

```diff
@@ -95,11 +103,24 @@ def __init__(self, port, ioloop=None, stop_when_source_finishes=False,
 
         """
         super(StreamServer, self).__init__(**kwargs)
-        logging.info('Initializing server...')
-        self.http_server = tornado.httpserver.HTTPServer(self,
+	if certfile == None or keyfile == None
```

Comment:
Error de sintaxis: falta el dos puntos del if.

Además, la comparación con None se debe hacer con "is": "if certfile is None or keyfile is None:"

Ya que estamos, te comento por si surge en el futuro que la comparación negativa con None se escribe con "is not" como en "if a is not None:"